dref/utils.py

```diff
@@ -107,719 +106,325 @@ def parse_type_of_onset(type):
     elif type == 'Sudden':
         type = Dref.OnsetType.SUDDEN
     elif type == 'Imminent':
-        type = Dref.OnsetType.Imminent
-    return type
-
-
-def parse_string_to_int(string):
-    char_to_check = ','
-    if string and char_to_check in string:
-        new_strings = string.split(',')
-        concat_string = new_strings[0] + new_strings[1]
-        return int(concat_string)
-    return int(string)
-
-
-def parse_boolean(string):
-    if string and string == 'Yes':
-        return True
-    elif string and string == 'No':
-        return False
+        type = Dref.OnsetType.IMMINENT
```

Comment:
Shadows built-in name 'type' – please use "onset_type" or similar, not the word type.